src/storage/src/source/mod.rs

```diff
@@ -750,9 +757,9 @@ where
             let upper_ts = resume_upper.as_option().copied().unwrap();
             let as_of = Antichain::from_elem(upper_ts.saturating_sub(1));
             let mut timestamper = match ReclockOperator::new(
-                persist_clients,
-                storage_metadata,
-                now,
+                persist_clients.clone(),
```

Comment:
Just to make it explicit, right? Because the runtime behavior is the same, as far I understand (I think I might have asked you about this some time ago)

Reply:
Yes, and (I think) yes :sweat_smile:. Plus, I'm quite sure we have a lint that disallows your code.

Reply:
I know we have that lint in persist, but did it ever land repo wide?